### PR TITLE
feat: localize about page

### LINKED ISSUE #453: localize the "About" content
In #444 , the translations for the home page were moved to our shared package. We're now ready to enable localization to the rest of the web package.

This will differ from other translations since this content is placed within a markdown file and used by both the web and server package.

## Translations

https://docs.google.com/document/d/1wSpEQX0k5ShGV2fsj5O9TM_Xq3k03UpT1A6Bh8sA8Xc/edit

## Content

https://github.com/CodeForBoulder/upswyng/blob/master/packages/common/src/content/About.md

## Checklist
- [ ] translations for both English and Spanish were added to the shared package
- [ ] translation keys were substituted for the correct web package content

## Out of Scope

Ensuring the language switching is supported in the server package.

### PR DESCRIPTION
**Pull request by jacobvenable**
This PR closes #453

## What does this PR do?

- converts translation json files to TS objects
- creates separate files for English and Spanish version of about page

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

oh

### CODE REVIEW

File: packages/common/src/content/about/es.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Spanish About UpSwyng
```
Comment (jacobvenable) [inline packages/common/src/content/about/es.md:1]: Still need to actually translate this 😅

File: packages/common/src/i18n/en.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
+import { AboutContent } from "@upswyng/common";
+
+export default {
+  about: {
+    content: AboutContent.en,
+  },
```
Comment (jacobvenable) [inline packages/common/src/i18n/en.ts:6]: Other than converting this file from json to TS, this was the only addition.

File: packages/server/src/routes/provider/about.svelte [hunk 3]
```diff
@@ -2,7 +2,7 @@
   import { AboutContent } from "@upswyng/common";
   import insane from "insane";
 
-  $: sanitizedAboutContent = insane(AboutContent, {
+  $: sanitizedAboutContent = insane(AboutContent.en, {
```
Comment (jacobvenable) [inline packages/server/src/routes/provider/about.svelte:5]: Only providing the English version to the provider portal for now.

File: packages/web/src/components/About.tsx [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+const About = () => {
+  const { i18n, t } = useTranslation("about");
+
+  const changeLng = (lng: string) => i18n.changeLanguage(lng);
+
+  return (
+    <Container>
+      <MuiHtml html={t("content")} />
+      <button onClick={() => changeLng("en")}>Switch en language</button>
+      <button onClick={() => changeLng("es")}>Switch es language</button>
```
Comment (jacobvenable) [inline packages/web/src/components/About.tsx:15]: Temp and will remove once ready for review 😛

**Review by jollyjerr — APPROVED** [review]
Yo Jacob, sorry for the late review. This looks great to me! I like the organization and the about page looks good 🚀 🎸 🌮

**Pull request merged**